src/items.rs

```diff
@@ -645,8 +645,14 @@ impl<'a> FmtVisitor<'a> {
         let mut items: Vec<_> = itemize_list_with(self.config.struct_variant_width());
 
         // If one of the variants use multiple lines, use multi-lined formatting for all variants.
-        fn is_multi_line_variant(item: &ListItem) -> bool {
+        let is_multi_line_variant = |item: &ListItem| -> bool {
             let variant_str = item.inner_as_ref();
+            if self.config.style_edition() < StyleEdition::Edition2024 {
```

Comment:
let's change this to `<= StyleEdition::Edition2021`. We've been using `>= StyleEdition::Edition2024 ` to indicate new formatting, and It'll be easier to grep for `Edition2021` to find all of the older formatting points.

Reply:
Thank you! Done in 56014626737e25e7d9a3e2e12f656231d5dce761.